lib/main.dart

```diff
@@ -27,8 +26,6 @@ class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
-      title: 'JustShareLah',
-      theme: AppTheme().buildThemeData(),
```

Comment:
why not use the Theme Data already in apptheme.dart? its a centralised place for our app's theme hahaha